pontoon/tags/templates/tags/includes/tags.html

```diff
@@ -0,0 +1,20 @@
+<div id="info-wrapper">
+
+{% import 'tags/widgets/tag_list.html' as TagList %}
+
+<div class="resources">
+
+  {{ TagList.header() }}
+  {% for tag in tags %}
+  {% set main_link = url('pontoon.tags.project.tag', project=project[0].slug, tag=tag.slug) %}
+  {% set chart_link = main_link %}
+  {% set latest_activity = tag.latest_activity %}
+  {% set chart = tag.chart %}
+  {{ TagList.item(tag, main_link, chart_link, latest_activity, chart) }}
+  {% endfor %}
+
+  {{ TagList.footer() }}
+</div>
+
+
+</div>
```

Comment:
This file should go into `projects/templates/projects/includes`. Each file in that folder represents a tab on the project dashboard.